sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_client_async.py

```diff
@@ -937,7 +983,8 @@ async def settle_messages_async(
         delivery_failed: Optional[bool] = None,
         undeliverable_here: Optional[bool] = None,
         message_annotations: Optional[Dict[Union[str, bytes], Any]] = None,
-        batchable: Optional[bool] = None
+        batchable: Optional[bool] = None,
+        **kwargs
```

Comment:
I don't care about this client too much as it's kinda deprecated - though we should avoid adding kwargs just to pass undocumented stuff around.